### PR TITLE
Profile Updated.

### PR DESCRIPTION
**Pull request by amit2020cs**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture, and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [x] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by elisemoe** [comment]
Hi @amit2020cs 👋😄 we can't wait to celebrate graduation with you! Be sure to submit the form before 12:00 AM PT today (in ~2.5hr) to be considered for swag. Your PR does not need to be merged before midnight.

File: _data/amit2020cs/amit2020cs.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Amit Kumar Singh 
+institution: Guru Govind Singh Indraprasth University 🚩
```
Comment (Mohammed-Shoaib) [inline _data/amit2020cs/amit2020cs.md:3]: Remove the 🚩(flag) as it is a non-ascii character.

File: _data/amit2020cs/amit2020cs.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Amit Kumar Singh 
+institution: Guru Govind Singh Indraprasth University
+profile_pic: Amit.jpg
```
Comment (Mohammed-Shoaib) [inline _data/amit2020cs/amit2020cs.md:4]: Git is case-sensitive. Could you please fix the file name? You could do one of the following:
1. Change `Amit.jpg` to `amit.jpg ` in the markdown file.
2. Rename `amit.jpg` to `Amit.jpg`.

File: _data/amit2020cs/amit2020cs.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+name: Amit Kumar Singh 
+institution: Guru Govind Singh Indraprasth University
```
Comment (Mohammed-Shoaib) [inline _data/amit2020cs/amit2020cs.md:3]: Possible typo for the name of institution?
From: Guru Govind Singh Indraprasth University
To: Guru Gobind Singh Indraprastha University

**Pull request merged**